baseapp-profiles/baseapp_profiles/managers.py

```diff
@@ -3,7 +3,7 @@
 
 class ProfileManager(models.Manager):
     def filter_user_profiles(self, user):
-        return self.filter(models.Q(members__user=user) | models.Q(owner=user))
+        return self.filter(models.Q(members__user=user) | models.Q(owner=user)).distinct()
```

Comment:
this distincts are not needed, only if we have "bad" data, where the same user is both owner and member, but I'm ok with adding them, don't hurt.